src/plays/schulte-tables/utils/shuffleArray.js

```diff
@@ -0,0 +1,15 @@
+const array = [
+  1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25
+];
+
+// declare the function
+const shuffle = () => {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+
+  return array;
+};
+
+export default shuffle;
```

Comment:
you can shorten the code. `Array.from` to create an array of numbers from 1 to 25 and then shuffle it using the [Fisher-Yates algorithm](https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle)

```javascript
const shuffle = () => {
  const array = Array.from({ length: 25 }, (_, i) => i + 1);

  for (let i = array.length - 1; i > 0; i--) {
    const j = Math.floor(Math.random() * (i + 1));
    [array[i], array[j]] = [array[j], array[i]];
  }

  return array;
};

export default shuffle;
```